src/Common/Integrations/Integration_Abstract.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Abstract for Integrations.
+ *
+ * @since   TBD
```

Comment:
```suggestion
 * @since TBD
```

Reply:
Needs a bulk search-replace to get rid of all this kind of extra spaces in multiple places.